node_modules/col-assets/lib/api.js

```diff
@@ -64,6 +67,13 @@ var getAssetProfile = module.exports.getAssetProfile = function(ctx, id, increme
 
     asset = asset.toJSON();
     asset.whiteboard_elements = _.pluck(asset.whiteboard_elements, ['element']);
+    asset.exported_whiteboards = _.pluck(asset.exported_whiteboards, ['id', 'title']);
```

Comment:
In fact we do need the `pluck`; because the association runs through whiteboard elements, the Sequelize call returns an array of whiteboard elements for each whiteboard, and specifying `attributes` below isn't sufficient to screen them out. We could remove the `attributes` option, but then the database will be returning many more unneeded asset-specific properties.